content/blog/kubecon-na-2022/index.md

```diff
@@ -0,0 +1,40 @@
+---
+path: '/kubecon-na-2022'
+title: 'Come meet us at Kubecon NA 2022'
+date: 2022-10-18T00:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'Announcements', 'Kubernetes']
+authors: ['Zain Asgar', 'Michelle Nguyen']
+emails: ['zasgar@pixielabs.ai', 'michelle@pixielabs.ai']
+---
+
+Pixie contributors will be attending this year’s [KubeCon + CloudNativeCon North America 2022](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/) in Detroit, Michigan!
+
+Come chat about open source observability for Kubernetes with Pixie’s core contributors at our booth in the [Project Pavilion](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/project-engagement/#project-pavilion).
+
+Pixie contributors will also be speaking at the following events:
+
+- [**OpenTelemetry or eBPF? That is the Question**](https://sched.co/1Auyh) ([eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/) - Monday, October 24, 2022 9:25 AM EDT)<br/><br/>
+_In the observability space, OpenTelemetry and eBPF are two technologies that have been rapidly changing the landscape. So which should you use? The OpenTelemetry project provides a rich set of tools with which teams can instrument their applications, enabling deep visibility into the application behavior. eBPF, on the other hand, has been powering instrumentation-less observability through other projects like the CNCF Pixie and Hubble projects. In this environment, users often wonder which approach they should turn to. In this session, we'll cover the strengths and weaknesses of both approaches, and show how both approaches have a role to play. We'll demonstrate how eBPF observability tools can be configured to export to OpenTelemetry collectors as automatic data sources. We'll then focus on the problem of tracing, and how request tracing works with eBPF and OpenTelemetry. In this process, we'll show how eBPF has the power to avoid some manual instrumentation; in contrast, we'll show how instrumentation is still required for true distributed tracing today. The session will wrap up with a perspective into the future of the two technologies, and what is on the horizon._
```

Comment:
The descriptions for our talks look like a wall of text in the preview. It might make sense to just summarize and make them shorter (2-3 sentences).